css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 100px;
```

Comment:
If you make the title `width: 100%`, this won't be necessary, since `text-align: center;` has already been applied